pjmedia/src/pjmedia/transport_srtp_dtls.c

```diff
@@ -949,6 +950,12 @@ static pj_status_t get_rem_addrs(dtls_srtp *ds,
 			  pj_sockaddr_get_addr_len(rem_rtp));
 	    }
 	}
+    } else {
+    	a = pjmedia_sdp_attr_find2(m_rem->attr_count, m_rem->attr,
+			       	   "rtcp-mux", NULL);
+	if (a) {
+	    pj_sockaddr_cp(rem_rtcp, rem_rtp);
+	}
```

Comment:
What if remote SDP has both attributes, rtcp-mux & rtcp? Shouldn't it check for rtcp-mux first?